### PR TITLE
Add Beautiful Cities of Morrowind main files

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Add location
Add load order
First step in "closing" this [issue](https://github.com/IllusiveMan196/morrowind/issues/2)
#52

### CODE REVIEW

**Comment by MacSplody** [comment]
> First step in "closing" this [issue](https://github.com/IllusiveMan196/morrowind/issues/2)

Files from the same mod page can be added in one pull request.
And all files from that modpage should be kept with the main file, unless they're replacement plugins for other mods.

**Comment by IllusiveMan196** [comment]
Well, this makes it way easier at least now. A moment, I've got those files added in local version already. Forgot alpahbetical oder of files... another edit?

File: masterlist.yaml [hunk 1]
```diff
@@ -523,6 +523,35 @@ plugins:
...
+      - name: 'multipatch.esp'
+      - name: 'BOM_pathgrid_reset.esp'
+
+  - name: 'Interior exterior flag reset.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
+      - name: 'BOM_pathgrid_reset.esp'
+      - name: 'BOM_OpenMW_plazas.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: We'll only add spaces if the files are from different mods.

```suggestion
  - name: 'Beautiful cities of Morrowind.ESP'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - 'Better Landscapes Stonewood Pass (RP Edit).esp'
      - 'Morrowind Anti-Cheese.esp'
  - name: 'BOM_pathgrid_reset.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - name: 'Beautiful cities of Morrowind.ESP'
      - name: 'multipatch.esp'
  - name: 'BOM_OpenMW_plazas.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - name: 'Beautiful cities of Morrowind.ESP'
      - name: 'multipatch.esp'
      - name: 'BOM_pathgrid_reset.esp'
  - name: 'Interior exterior flag reset.esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
    after:
      - name: 'Beautiful cities of Morrowind.ESP'
      - name: 'multipatch.esp'
      - name: 'BOM_pathgrid_reset.esp'
      - name: 'BOM_OpenMW_plazas.esp'
```

**Comment by MacSplody** [comment]
@IllusiveMan196 Are you happy with that? It can be merged if it's done.
Though you could make one or two format changes.

File: masterlist.yaml [hunk 2]
```diff
@@ -523,6 +523,32 @@ plugins:
...
+  - name: 'Beautiful cities of Morrowind.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - 'Better Landscapes Stonewood Pass (RP Edit).esp'
+      - 'Morrowind Anti-Cheese.esp'
+  - name: 'BOM_pathgrid_reset.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
    after:
      - 'Beautiful cities of Morrowind.ESP'
      - 'multipatch.esp'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -523,6 +523,32 @@ plugins:
...
+  - name: 'BOM_OpenMW_plazas.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
+      - name: 'BOM_pathgrid_reset.esp'
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
      - 'Beautiful cities of Morrowind.ESP'
      - 'BOM_pathgrid_reset.esp'
      - 'multipatch.esp'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -523,6 +523,32 @@ plugins:
...
+  - name: 'Interior exterior flag reset.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/49231/' ]
+    after:
+      - name: 'Beautiful cities of Morrowind.ESP'
+      - name: 'multipatch.esp'
+      - name: 'BOM_pathgrid_reset.esp'
+      - name: 'BOM_OpenMW_plazas.esp'
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
      - 'Beautiful cities of Morrowind.ESP'
      - 'BOM_OpenMW_plazas.esp'
      - 'BOM_pathgrid_reset.esp'
      - 'multipatch.esp'
```

**Review by MacSplody — CHANGES_REQUESTED** [review]
Just a few format changes required.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@IllusiveMan196 Thanks.

**Pull request merged**